### PR TITLE
ensure domains can be wildcards

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

File: project/templates/configs/cms.nginx.conf [hunk 1]
```diff
@@ -1,9 +1,9 @@
 server {
     listen 80;
     {% if deploy_environment == 'qa' %}
-    server_name cms.{{country}}.{{deploy_environment}}.{{app_type}}.unicore.io;
+    server_name cms.{{country}}.{{deploy_environment}}.{{app_type}}.qa-hub.unicore.io;
```
Comment (smn) [inline project/templates/configs/cms.nginx.conf]: means we can throw the `deploy_environment` bit away

Comment (miltontony) [inline project/templates/configs/cms.nginx.conf]: @smn not entirely - the `-` will still be a problem

Comment (miltontony) [inline project/templates/configs/cms.nginx.conf]: @smn unless the `deploy_env` is set to `qa-hub`

**Comment by miltontony** [comment]
@smn ready again

**Comment by smn** [comment]
:+1: but future PRs could change `deploy_environment` to just be a Hub name.

**Pull request merged**